### PR TITLE
Render and document components to styleguide

### LINKED ISSUE #59: Generate component style guide using styleguidist middleware for rendering components out of application

### PR DESCRIPTION
**Pull request by eliperelman**
Fixes #59.

### CODE REVIEW

**Comment by eliperelman** [comment]
@helfi92 did you run `yarn` in the repo?

**Comment by helfi92** [comment]
@eliperelman Yes, I did. I will try to update yarn to the latest version and see if that fixes it.

File: src/components/DateDistance/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Default usage:
```
Comment (helfi92) [inline src/components/DateDistance/README.md]: Are you able to add some margin in between the "View Code" tab and the code portion globally? I find it hard to see the tab's bottom border.

screen shot 2018-04-16 at 11 08 31 am

File: src/components/FontStager/index.jsx [hunk 2]
```diff
@@ -2,6 +2,10 @@ import { Component } from 'react';
 import FoutStager from 'react-fout-stager';
 import './index.css';
 
+/**
+ * Responsible for loading the application typefaces progressively
+ * using FOUT stage techniques.
+ */
 export default class FontStager extends Component {
```
Comment (helfi92) [inline src/components/FontStager/index.jsx]: Could we have a dark colour background with white/gray text for code segments?

screen shot 2018-04-16 at 11 09 14 am

File: src/components/DateDistance/README.md (hunk 1 above)
Comment (eliperelman) [reply src/components/DateDistance/README.md]: screen shot 2018-04-16 at 4 15 21 pm

File: src/components/FontStager/index.jsx (hunk 2 above)
Comment (eliperelman) [reply src/components/FontStager/index.jsx]: screen shot 2018-04-16 at 4 15 32 pm

**Pull request merged**